plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -122,7 +124,6 @@ export const WorkflowInstancePageContent: React.FC<{
           <WorkflowResult
             assessedInstance={assessedInstance}
             className={styles.middleRowCard}
-            cardClassName={styles.autoOverflow}
```

Comment:
are you sure we don't need the "overflow: auto" also for the cardClassName? What if the outputs are a long list? Does it work without it?
And if not, since WorkflwoResult is only called by one place, might as well remove the property all together

Reply:
By the other change in this PR, we newly scroll whole card content, not just the inner part. It's better than having multiple scrollbars on a small place. 
I can remove the property for all. On the other hand, there is no harm in keeping it.

Reply:
I checked myself, it must have overflow: auto only in className as you did and it works well.
Please remove cardClassNameProperty instead of making it optional